go.mod

```diff
@@ -24,7 +24,11 @@ require (
 	sigs.k8s.io/controller-runtime v0.12.2
 )
 
-require github.com/google/go-cmp v0.5.9
+require (
+	github.com/backube/volsync v0.4.0
```

Comment:
not too important we can follow up to bump this up to 0.7 later. This is only used to add to scheme to list them if test fails.
Not using any struct fields.

Reply:
We don't have to change the volsync version. I removed ocp 4.8 and 4.9 from master.

Reply:
The volsync version problem is not the go.mod problem.. but the subscription channel in yaml file.